config.go

```diff
@@ -95,15 +95,29 @@ type Config struct {
 
 	// File logger options
 	FileLogger *FileLoggerConfig `mapstructure:"file_logger_options"`
+
+	// UseLocalTimeEncoder is used to set the encoder to use local time instead of UTC.
+	UseLocalTimeEncoder bool `mapstructure:"use_local_time_encoder"`
+
+	// ShowCaller is used to set the encoder to show the caller.
+	ShowCaller bool `mapstructure:"show_caller"`
```

Comment:
> When developing custom plugins, it was difficult to find the corresponding location in the log, so caller was added
In the production environment, the log time format is not readable enough, so the local time display function is added

I'm not sure if I understand this. The caller is the line where your log is. How is that difficult to find where you send this log?

Reply:
When developing custom plugins, although logs show what happened, pinpointing the exact source of a log entry in complex plugin code can be challenging without caller information. By enabling show_caller, we can immediately see which file and line generated the log, significantly improving debugging efficiency.

Reply:
How can it be complex? It is just ctrl+f (or / for vim) -> enter log message -> here you are.